docs/getting_started/troubleshooting.rst

```diff
@@ -11,7 +11,24 @@ Frequently encountered issues
 This is a list of the most frequently encountered issues when using Modin. Some of these
 are working as intended, while others are known bugs that are being actively worked on.
 
-Error During execution: ``ArrowIOError: Broken Pipe``
+
+
+Error during execution: ``NotImplementedError``
+-----------------------------------------------
+
+Please note, that while Modin covers a large portion of the pandas API, not all functionality is implemented. For some methods, you may see the following:
+
+.. code-block:: text
+
+  NotImplementedError: To contribute to Modin, please visit github.com/modin-project/modin.
+
+To understand which function will lead to this error, we have compiled a list of :doc:`currently supported methods </supported_apis/index>`. When you see this error, Modin defaults to pandas by converting the Modin dataframe to pandas to perform the operation. Once the operation is complete in pandas, it is coverted back to a Modin dataframe. These operations will have a high overhead due to the communication involved and will take longer than pandas. When this is happening, a warning will be given to the user to inform them that this operation will take longer than usual. 
```

Comment:
If a user sees NotImplementedError, Modin won't be defaulting to pandas.

Reply:
fixed to correct warning message